weblate/templates/snippets/suggestions.html

```diff
@@ -25,7 +25,7 @@
         {% else %}
         {% perm 'suggestion.delete' suggestion as user_can_delete_suggestion %}
         {% if user_can_vote_suggestion or user_can_accept_suggestion or user_can_delete_suggestion %}
-        <div class="btn-float pull-right flip">
+        <div class="btn-float pull-right flip" style="top: -20px">
```

Comment:
Please avoid inline styles.

Reply:
yeah, I don't like it too, I wonder what should I do instead? Changing the original class doesn't sound like a good idea either, I guess I could create new one?